builder/words

```diff
@@ -0,0 +1,42 @@
+hello                    |     hello
```

Comment:
No, i'm wondering if there's a reason this code exists in the builder instead of the parser. Is this a runtime evaluation thing?

Sorry if I'm behind; it's been a very crazy month.

-Erik

> On Mar 20, 2015, at 7:03 PM, Doug Davis <notifications@github.com> wrote:
> 
> In builder/words <https://github.com/docker/docker/pull/10431#discussion_r26887661>:
> 
> > @@ -0,0 +1,42 @@
> > +hello                    |     hello
> @erikh <https://github.com/erikh> I assume you mean to add a "''" test, right? if so, done.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/docker/docker/pull/10431/files#r26887661>.
>